ui/admin/app/components/form/credential-library/index.js

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
+import { options } from 'api/models/credential-library';
+import { computed } from '@ember/object';
+
+export default class FormCredentialLibraryIndexComponent extends Component {
+  // =properties
+  /**
+   * @type {object}
+   */
+  httpMethods = options.http_methods;
+
+  /**
+   * Only allow HTTP request body field if http_method is set to POST.
+   * @type {boolean}
+   */
+  @computed('args.model.attributes.http_method', 'isSaving')
+  get cannotSaveHttpRequestBody() {
```

Comment:
This is specifically for the disabled state, not save-ability per se.

Reply:
Good point. If model save state isn't checked in this, is there another way the radiobutton group can be disabled during save?
Or is your comment referring to the name of the function?

Reply:
Refactored this into `isHttpRequestBodyAllowed` without using `isSaving` method. Reflects design change where text area is only shown when http_method value is POST.